From ada74fb0ca3099f33d173eb664bd7e42025a3277 Mon Sep 17 00:00:00 2001
From: Prabhu Sundararaj <prabhu.sundararaj@nxp.com>
Date: Mon, 9 Dec 2024 09:15:11 +0800
Subject: [PATCH 1/2] MGS-6801 [#ccc] vkmark on wayland

Extend the wayland surface properties with VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR

Upstream-Status: Inappropriate [i.MX-specific]
Signed-off-by: Prabhu Sundararaj <prabhu.sundararaj@nxp.com>
Signed-off-by: Jiyu Yang <jiyu.yang@nxp.com>
---
 wsi/wayland/surface_properties.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wsi/wayland/surface_properties.cpp b/wsi/wayland/surface_properties.cpp
index e6435b9..bc1a737 100644
--- a/wsi/wayland/surface_properties.cpp
+++ b/wsi/wayland/surface_properties.cpp
@@ -89,7 +89,7 @@ VkResult surface_properties::get_surface_capabilities(VkPhysicalDevice physical_
 
    /* Composite alpha */
    pSurfaceCapabilities->supportedCompositeAlpha = static_cast<VkCompositeAlphaFlagBitsKHR>(
-      VK_COMPOSITE_ALPHA_PRE_MULTIPLIED_BIT_KHR | VK_COMPOSITE_ALPHA_INHERIT_BIT_KHR);
+      VK_COMPOSITE_ALPHA_PRE_MULTIPLIED_BIT_KHR | VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR);
    return VK_SUCCESS;
 }
 
-- 
2.37.1

